src/klicky.ts

```diff
@@ -1,3 +1,8 @@
 export const Klicky = () => {
   console.log("klickly ran!");
 };
+
+
+document.getElementById("yellow")?.addEventListener("click", function(){
```

Comment:
Nice job adding an event listener for this element! Can you try adding a different event to the page, perhaps something like `mousedown` or `mouseup`?